exchanges/interfaces.go

```diff
@@ -98,6 +98,9 @@ type IBotExchange interface {
 	UpdateOrderExecutionLimits(ctx context.Context, a asset.Item) error
 
 	AccountManagement
+
+	// GetServerTime returns the current exchange server time.
+	GetServerTime(ctx context.Context, ai asset.Item) (time.Time, error)
```

Comment:
I don't think this doesn't need its own section and commentary, please add to the main chunk of functions.